src/CalcManager/Ratpack/conv.cpp

```diff
@@ -373,7 +373,7 @@ PNUMBER nRadixxtonum( _In_ PNUMBER a, uint32_t radix, int32_t precision)
 //
 //-----------------------------------------------------------------------------
 
-PNUMBER numtonRadixx(_In_ PNUMBER a, uint32_t radix)
+PNUMBER numtonRadixx(PNUMBER a, uint32_t radix)
```

Comment:
I'm surprised by this change, was this intended?

Reply:
Honestly, I'm not 100% sure, but I think it was because `a` is actually modified.

Reply:
Wouldn't that make it `_Inout_`?

Reply:
In this codebase, I think they intend to treat `PNUMBER` and such as conceptually `_In_` arguments, since they usually pass parameters that are conceptually modified as pointer-to-pointer (`PNUMBER*`) or reference (`PNUMBER&`). I'm going to recompile with this as an `_In_` argument and see if it complains.

Reply:
Doesn't seem to complain, will return to original.